aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -83,6 +89,34 @@ private AWSDataStorePlugin(
             @NonNull ModelSchemaRegistry modelSchemaRegistry,
             @NonNull ApiCategory api,
             @Nullable DataStoreConfiguration userProvidedConfiguration) {
+        // See https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling
+        // Without this, the host app crashes on an unhandled exception.
+        RxJavaPlugins.setErrorHandler(e -> {
```

Comment:
All fair points. I guess my thinking was that we wouldn't want to be the ones crashing the app because of an unhandled exception. I'm definitely not crazy about the pattern myself. I like the idea of perhaps passing something in. Let me circle back on this.

Reply:
removed this guy too.